scripts/postprocess.py

```diff
@@ -12,8 +12,21 @@
 sns.set_theme()
 
 # plt_type : ppn | time
-def _gen_nodes_plot(plt_type, title, ylabel, file_name, df, logx, logy=False, combine_mb=False, filts=None, replaces=None):
-    fig, ax = plt.subplots()
+def _gen_nodes_plot(
```

Comment:
As input parameters increased (and their value has to follow some constrains) I would improve the doc.

Reply:
Can you give it a quick check, please?